README.md

```diff
@@ -1 +1,21 @@
-# Social-network
\ No newline at end of file
+# We are A-Team
+
+## The user stories
+
+1. As user, I want to be able to sign up and have my own account, so that my chats are kept
```

Comment:
what about replacing "kept" with "saved and kept in the database?

Reply:
the user doesn't know about the database

Reply:
but you are right that the stories are for us and I comply